Lib/test/test_call.py

```diff
@@ -131,6 +131,10 @@ def test_varargs2(self):
         msg = r"__contains__\(\) takes exactly one argument \(2 given\)"
         self.assertRaisesRegex(TypeError, msg, {}.__contains__, 0, 1)
 
+    def test_varargs3(self):
+        msg = r"from_bytes\(\) takes at most 2 positional arguments \(3 given\)"
+        self.assertRaisesRegex(TypeError, msg, int.from_bytes, b'a', 'little', False)
```

Comment:
This unit test is wrong: it doesn't catch the fixed bug. You need to add "^" at the start of the regex.

You may add "^" and "$" to all regex of this file ;-) (Maybe even write an helper function to create the regex?)